components/script/dom/bindings/root.rs

```diff
@@ -78,6 +78,16 @@ where
 
 /// Represents values that can be rooted through a stable address that will
 /// not change for their whole lifetime.
+///
+///
+/// /// `StableTraceObject` is an unsafe trait that requires implementors to ensure certain safety guarantees.
```

Comment:
```suggestion
/// /// `StableTraceObject` is an unsafe trait that requires implementors to ensure certain safety guarantees.
```

There are two levels of rustdoc comments here.